Documentation/Usage.md

```diff
@@ -115,204 +115,254 @@ url.
 
+```swift
 let package = Package(
-    name: "Clibgit",
-    pkgConfig: "libgit2"
+    name: "example",
+    targets: [
+        .systemLibrary(name: "cairo", pkgConfig: "cairo"),
+        .target(name: "example", dependencies: ["cairo"])
+    ]
 )
 ```
 
-The `pkgConfig` parameter helps SwiftPM in figuring out the include and library
-search paths for the system library.  Note: If you don't want to use the `pkgConfig`
-parameter you can pass the path of a directory containing the library using the
-`-L` flag in commandline when building your app:
+The `pkgConfig:` parameter specifies the name of the system package that the package manager will ask the system `pkg-config` tool about. Without this information, `clang` may not know where to look for the installed Cairo headers. You can see what `pkg-config` will tell the package manager by running the tool directly in the terminal:
 
-    example$ swift build -Xlinker -L/usr/local/lib/
+```bash
+$ pkg-config --cflags cairo
+```
+```
+-I/usr/include/cairo -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include 
+-I/usr/include/pixman-1 -I/usr/include/freetype2 -I/usr/include/libpng16 
+-I/usr/include/freetype2 -I/usr/include/libpng16
+```
 
-Edit `module.modulemap` so it consists of the following:
+Unlike Swift targets, the `name:` parameter in a system library target is only used by the package manager; the name of the module as seen by Swift is entirely determined by the `module cairo` declaration in the `module.modulemap` file. You are free to use a different name for the module within the package description:
 
-    module Clibgit [system] {
-      header "/usr/local/include/git2.h"
-      link "git2"
-      export *
-    }
+```swift
+let package = Package(
+    name: "example",
+    targets: [
+        // by default the package manager assumes the module lives in a folder of the 
+        // same name underneath the `Sources/` directory
+        .systemLibrary(name: "Foo", path: "Sources/cairo", pkgConfig: "cairo"),
+        .target(name: "example", dependencies: ["Foo"])
+    ]
+)
+```
 
-> The convention we hope the community will adopt is to prefix such modules
-> with `C` and to camelcase the modules as per Swift module name conventions.
-> Then the community is free to name another module simply `libgit` which
-> contains more “Swifty” function wrappers around the raw C interface.
+In our `main.swift` we can import and use the system module like any other module:
 
-Packages are Git repositories, tagged with semantic versions, containing a
-`Package.swift` file at their root.  Initializing the package created a
-`Package.swift` file, but to make it a usable package we need to initialize a
-Git repository with at least one version tag:
+```swift
+import cairo
 
-    Clibgit$ git init
-    Clibgit$ git add .
-    Clibgit$ git commit -m "Initial Commit"
-    Clibgit$ git tag 1.0.0
+let surface:OpaquePointer = cairo_image_surface_create(CAIRO_FORMAT_ARGB32, 120, 120)
```

Comment:
nit: a space between `:` and `OpaquePointer`, or just omit the type the compiler should be able to infer (I hope).